examples/neural_compressor/language-modeling/run_clm.py

```diff
@@ -677,19 +714,24 @@ def compute_metrics(eval_preds):
         trainer.save_metrics("train", metrics)
         trainer.save_state()
 
-    if optim_args.apply_quantization and optim_args.quantization_approach in {"static", "dynamic"}:
+    if optim_args.apply_quantization and optim_args.quantization_approach in {"static", "dynamic", "weight_only"}:
         model = trainer.model if isinstance(trainer.model, PreTrainedModel) else trainer.model._model
+        model.seqlen = block_size

```

Comment:
Why is it used for ? Can we include it in the config or quantizer directly instead of modifying the original model ?

Reply:
Of course we can add an enhancement in INC to move this feature to config/quantizer. This parameter is used to check calibration data length to ensure they won't exceed model's config acceptable sequence length, since we do not know how users tokenize their own datasets.

Reply:
Ok I see, then it could make sense to check this in the examples scripts directly, I don't think we need to add this verification in the `INCQuantizer` or in `neural-compressor` (in `GPTQuantizer`) as it belongs to a different scope in my opinion

Reply:
Once this point is fixed we can merge in my opinion!